lib/wraft_doc_web/controllers/content_type_controller.ex

```diff
@@ -591,6 +592,7 @@ defmodule WraftDocWeb.Api.V1.ContentTypeController do
          %Theme{} <- Document.get_theme(theme_id, current_user),
          %ContentType{} = content_type <-
            Document.create_content_type(current_user, params) do
+      Typesense.create_document(content_type, content_type)
```

Comment:
Having the indexing operation here can increase our APIs latency. So we should think of a way to handle this in the backround. Keep a note of this in the backlog